.github/workflows/udr-migration-check.yml

```diff
@@ -0,0 +1,202 @@
+name: UDR Migration Link Check
```

Comment:
@Younique98 Let's rename this to `UDR Product Link Check`, as I can imagine this will be used more then just during migrations.